.env

```diff
@@ -0,0 +1,2 @@
+CORE_API_KEY=
+CORE_RECOMMENDER_API_KEY=
```

Comment:
I would rather pass environment dependent variables through `export` and catch it from `process.env.NODE_ENV`

Reply:
This is mostly for local development only. IMHO it's better than exporting because a user may forgot about it.  On production, the variable should be exported. For example, I already did it for [now via secrets](https://zeit.co/docs/v2/build-step/#using-environment-variables-and-secrets).